Extensions/probing.hh

```diff
@@ -281,15 +280,26 @@ inline double getReactivityScore(const Subsequence &leftBase, const bool isUnpai
 		    }
 		    infile.close();
 		}
-		if (probingData.size() < (leftBase.seq->n)) {
-			std::cerr << "Warning: chemical probing data file '" << getProbing_dataFilename() << "' misses " << (leftBase.seq->n - probingData.size()) << " data-row(s) " << std::endl << "         compared to the number of nucleotides in your input sequence." << std::endl << "         Missing values will be set to 0.0!" << std::endl;
+        
+        // needs reworking when changing the way the whitespace between two sequences is represented
```

Comment:
can you add a `TODO (kmaibach): ` to the comment? Some IDEs do recognize these annotations and flag them accordingly